.github/workflows/run-tck.yaml

```diff
@@ -10,11 +10,13 @@ on:
 env:
   # TODO once we have the TCK for 0.4.0 we will need to look at the branch to decide which TCK version to run.
   # Tag of the TCK
-  TCK_VERSION: 0.3.0.beta3
+  TCK_VERSION: 601bf0d6f63baafb079c2ab23f53d73ce03e5461
```

Comment:
Let's add a comment here why it was done, maybe referencing this PR comment(s) with our discussion?